.gitignore

```diff
@@ -50,6 +50,8 @@ coverage.xml
 *.cover
 .hypothesis/
 .pytest_cache/
+.ruff_cache/
+bigquery.json
```

Comment:
the bigquery.json should be removed I guess?

Reply:
The file is not there in this PR and it should be part of gitignore